projects/storefrontlib/src/lib/ui/pages/order-confirmation-page/order-confirmation-page.component.html

```diff
@@ -1,3 +1,5 @@
 <cx-page-layout>
-  <cx-order-confirmation></cx-order-confirmation>
+  <ng-template cxOutletRef="OrderConfirmationOverviewComponent">
```

Comment:
can we add a TODO that we should move this into a cms driven component?

Reply:
Done